AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_OcclusionCullingPlaneAdded.py

```diff
@@ -0,0 +1,159 @@
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+"""
+
+class Tests:
+    creation_undo = (
+        "UNDO Entity creation success",
+        "UNDO Entity creation failed")
+    creation_redo = (
+        "REDO Entity creation success",
+        "REDO Entity creation failed")
+    occlusion_culling_plane_entity_creation = (
+        "Occlusion Culling Plane Entity successfully created",
+        "Occlusion Culling Plane Entity failed to be created")
+    occlusion_culling_plane_component_added = (
+        "Entity has a Occlusion Culling Plane component",
+        "Entity failed to find Occlusion Culling Plane component")
+    enter_game_mode = (
+        "Entered game mode",
+        "Failed to enter game mode")
+    exit_game_mode = (
+        "Exited game mode",
+        "Couldn't exit game mode")
+    is_visible = (
+        "Entity is visible",
+        "Entity was not visible")
+    is_hidden = (
+        "Entity is hidden",
+        "Entity was not hidden")
+    entity_deleted = (
+        "Entity deleted",
+        "Entity was not deleted")
+    deletion_undo = (
+        "UNDO deletion success",
+        "UNDO deletion failed")
+    deletion_redo = (
+        "REDO deletion success",
+        "REDO deletion failed")
+
+
+def AtomEditorComponents_OcclusionCullingPlane_AddedToEntity():
+    """
+    Summary:
+    Tests the occlusion culling plane component can be added to an entity and has the expected functionality.
+
+    Test setup:
+    - Wait for Editor idle loop.
+    - Open the "Base" level.
+
+    Expected Behavior:
+    The component can be added, used in game mode, hidden/shown, deleted, and has accurate required components.
+    Creation and deletion undo/redo should also work.
+
+    Test Steps:
+    1) Create a Occlusion Culling Plane entity with no components.
+    2) Add a Occlusion Culling Plane component to Occlusion Culling Plane entity.
+    3) UNDO the entity creation and component addition.
+    4) REDO the entity creation and component addition.
+    5) Enter/Exit game mode.
+    6) Test IsHidden.
+    7) Test IsVisible.
+    8) Delete Occlusion Culling Plane entity.
+    9) UNDO deletion.
+    10) REDO deletion.
+    11) Look for errors.
+
+    :return: None
+    """
+
+    import azlmbr.legacy.general as general
+
+    from editor_python_test_tools.editor_entity_utils import EditorEntity
+    from editor_python_test_tools.utils import Report, Tracer, TestHelper
+    from Atom.atom_utils.atom_constants import AtomComponentProperties
+
+    with Tracer() as error_tracer:
+        # Test setup begins.
+        # Setup: Wait for Editor idle loop before executing Python hydra scripts then open "Base" level.
+        TestHelper.init_idle()
+        TestHelper.open_level("", "Base")
+
+        # Test steps begin.
+        # 1. Create a occlusion culling plane entity with no components.
+        occlusion_culling_plane_entity = EditorEntity.\
```

Comment:
instead of using `\` for a new line just make new lines where `(` occurs in the code, or add `( )` where none exists.

For this particular example it would be:
```
occlusion_culling_plane_entity = EditorEntity.create_editor_entity(
AtomComponentProperties.occlusion_culling_plane())
```

Reply:
Fixed!